### PR TITLE
Adds main branch to eland repo to docs build

### PR DESCRIPTION
**Pull request by sethmlarson**
## Overview

This PR adds a main branch to the `eland` repo in the conf.yml file of the docs build system.

### Note

**It cannot be merged until _after_ a "main" branch is created in the eland repo.**

### CODE REVIEW

File: conf.yaml [hunk 1]
```diff
@@ -581,7 +581,7 @@ contents:
               - title:      eland
                 prefix:     eland
                 current:    master
-                branches:   [ master ]
+                branches:   [ {main: master} ]
                 live:       [ master ]
```
Comment (szabosteve) [inline conf.yaml:585]:
```suggestion
                live:       [ main ]
```

**Review by szabosteve — COMMENTED** [review]
I think we also need to change the value of `live` to `main`.

**Comment by sethmlarson** [comment]
Thanks @szabosteve!

**Review by szabosteve — APPROVED** [review]
LGTM!

**Review by gtback — APPROVED** [review]

**Comment by sethmlarson** [comment]
@szabosteve @gtback I've renamed the branch in GitHub now that infra has merged their PR. This should be good to go!

**Comment by gtback** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by szabosteve** [comment]
@lcawl Thank you for handling this!

**Pull request merged**